### PR TITLE
Update @flowio/lib-apidoc and regenerate

### PR DESCRIPTION
**Pull request by benwaffle**
Needed for https://github.com/flowcommerce/console/pull/1604

### CODE REVIEW

**Review by RoryH — APPROVED** [review]
LGTM, but judging by the changes, it's been a while since we updated 😬

**Comment by benwaffle** [comment]
We might want to have github hide diffs for autogenerated files:

.gitattributes:
```
src/** linguist-generated
docs/** linguist-generated
```

File: src/clients/client.js [hunk 1]
```diff
@@ -120,7 +120,17 @@ export default class Client extends EventEmitter {
...
+    let body;
+    if (opts.body != null) {
+      if (Buffer.isBuffer(opts.body))
+        body = opts.body;
+      else if (typeof opts.body === 'string')
+        body = opts.body;
+      else
+        body = JSON.stringify(opts.body);
+    }
+
```
Comment (benwaffle) [inline src/clients/client.js:133]: this is the critical change i want to pull in

**Review by chrhicks — APPROVED** [review]
LGTM

**Pull request merged**